doc/kuksa_analysis/kuksa_requirementss.md

```diff
@@ -0,0 +1,202 @@
+# Rerequirements
+
+# Content
+- [Rerequirements](#rerequirements)
+- [Content](#content)
+- [Requirements status](#requirements-status)
+- [Functional requirements](#functional-requirements)
+  - [As Signal Consumer](#as-signal-consumer)
+    - [FR1-ASC](#fr1-asc)
+    - [FR2-ASC](#fr2-asc)
+    - [FR3-ASC](#fr3-asc)
+    - [FR4-ASC](#fr4-asc)
+    - [FR5-ASC](#fr5-asc)
+    - [FR6-ASC](#fr6-asc)
+    - [FR7-ASC](#fr7-asc)
+    - [FR8-ASC](#fr8-asc)
+  - [As Databroker](#as-databroker)
+    - [FR1-AD](#fr1-ad)
+    - [FR2-AD](#fr2-ad)
+    - [FR3-AD](#fr3-ad)
+    - [FR4-AD](#fr4-ad)
+  - [As Provider](#as-provider)
+    - [FR1-AP](#fr1-ap)
+    - [FR2-AP](#fr2-ap)
+    - [FR3-AP](#fr3-ap)
+- [Non-Functional Requirements](#non-functional-requirements)
+- [Domain requirements](#domain-requirements)
+
+# Requirements status
+| Description                               | Status                       |
+|-------------------------------------------|------------------------------|
+| Implemented and Verified                  | 🟢                           |
+| Approved, Not Yet Implemented             | 🟡                           |
+| Long Term Goal                            | 🔴                           |
+
+# Functional requirements
+
+## As Signal Consumer
+### FR1-ASC
+    Title: Single Access Service Point for Signal Consumer
+
+    Status: 🟡
+
+    Description: The Signal Consumer shall have a single service point accessible from my execution environment.
+   
+### FR2-ASC
+    Title: Uniform Retrieval of Vehicle and Environmental Signal Values
+
+    Status: 🟡
+
+    Description: The Signal Consumer shall get sensor values or actuator values of the vehicle signals and its environment(air temperature, etc) in a uniform manner.
+      1.  IF there is an error
+          * THEN the Signal Consumer shall not receive any signal value, just one single error with detailed error information
+          * ELSE the Signal Consumer shall receive exactly all the signals’ states(failure or value) requested.
+
+          n. A signal can have value out of the set of the defined value restriction/data type but it can also have those additional failure states:
```

Comment:
```suggestion
          n. A signal can have a value in range of the defined restrictions/data type. Additionally it can hold these failure states:
```